packages/create-astro/src/index.ts

```diff
@@ -118,7 +118,7 @@ export async function main() {
 
 	// Don't touch the template name if a GitHub repo was provided, ex: `--template cassidoo/shopify-react-astro`
 	const isThirdParty = options.template.includes('/');
-	const templateTarget = options.template.includes('/')
+	const templateTarget = isThirdParty
```

Comment:
I think this should be in another PR, just so you don't have to do a changeset unnecessarily

Reply:
Reverted. PR here: #4605